installer/cloud-sql-proxy-operator.yaml

```diff
@@ -1,4 +1,4 @@
-# Copyright 2022 Google LLC
```

Comment:
This file is generated. Changes in this file reflect the changes to godoc in `authproxyworkload_types.go`